crates/smb-transport/README.rdma.md

```
@@ -0,0 +1,18 @@
+# About RDMA & SMB
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)